apps/liteprotocoltester/README.md

```diff
@@ -205,7 +194,18 @@ docker build -t liteprotocoltester:latest -f Dockerfile.liteprotocoltester ../..
 
 # edit and adjust .env file to your needs and for the network configuration
 
-docker run --env-file .env liteprotocoltester:latest RECEIVER <service-node-ip4-peer-address>
+docker run --env-file .env liteprotocoltester:latest RECEIVER <service-node-peer-address>
```

Comment:
I think it'd help to have an example on each command so that an not-experienced reader can learn about the expected address format (that tiny suggestion also applies to the following lines as well.)